src/main/resources/swagger/api-docs.yaml

```diff
@@ -2734,7 +2763,8 @@ paths:
     get:
       tags:
         - Profile
-      summary: List billing projects for a user
+      summary: List billing projects for a user. Deprecated, use v2 billing API.
+      deprecated: true
```

Comment:
These "profile" methods actually dispatch to the `user/billing` (v1) endpoints.

![image](https://user-images.githubusercontent.com/484484/192309068-67a6cf6a-eada-4982-b4fa-6bf33ad5230a.png)